x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -69,31 +113,32 @@ export const CaseViewTabs = React.memo<CaseViewTabsProps>(({ caseData, activeTab
             {
               id: CASE_VIEW_PAGE_TABS.ALERTS,
               name: (
-                <>
-                  {ALERTS_TAB}
-                  {features.alerts.isExperimental ? (
-                    <ExperimentalBadge
-                      label={EXPERIMENTAL_LABEL}
-                      size="s"
-                      iconType="beaker"
-                      tooltipContent={EXPERIMENTAL_DESC}
-                      tooltipPosition="bottom"
-                      data-test-subj="case-view-alerts-table-experimental-badge"
-                    />
-                  ) : null}
-                </>
+                <AlertsTab
+                  isLoading={isAlertLoading}
+                  isExperimental={features.alerts.isExperimental}
+                  totalAlerts={alertStatsData?.metrics.alerts?.count}
```

Comment:
Could you please update the tests in`x-pack/plugins/cases/public/components/case_view/case_view_tabs.tests.tsx` ?